src/actions/sources/loadSourceText.js

```diff
@@ -15,14 +16,12 @@ import type { ThunkArgs } from "../types";
 const requests = new Map();
 
 async function loadSource(source: Source, { sourceMaps, client }) {
-  if (sourceMaps.isOriginalId(source.id)) {
-    return await sourceMaps.getOriginalSourceText(source);
+  if (isOriginalId(source.id)) {
+    return sourceMaps.getOriginalSourceText(source);
```

Comment:
why is `sourceMaps` used here and passed into functions in other places? if we  are importing `devtools-source-map` already at the top, could we not just pull in all the other functions
e.g `import { isOriginalId, getOriginalSourceText } from "devtools-source-map";` or am i missing something?

Reply:
ahhh ..its for the workers, right?

Reply:
yeah - and the source map worker is different from the others because it is shared w/ the other panels

Reply:
cool thanks!